check_memory.py

```diff
@@ -0,0 +1,43 @@
+# Script that tries to prove that the quickjs wrapper does not leak memory.
+#
+# It finds the leak if a Py_DECREF is commented out in module.c.
+
+import gc
+import tracemalloc
+import unittest
+
+import quickjs
+import test_quickjs
+
+def run():
+    loader = unittest.TestLoader()
+    suite = loader.discover(".")
+    runner = unittest.TextTestRunner()
+    runner.run(suite)
+
+def main():
+    print("Warming up (to discount regex cache etc.)")
+    run()
+
+    tracemalloc.start(25)

```

Comment:
Are we sure that 25 frames are enough to capture any snapshot differences?

Reply:
It is just a limit of the tracebacks displayed for memory allocations. If it is not enough, the differences will still be displayed, but the tracktrace may be too short to see the whole path up to where the allocation took place.

Reply:
Ok. Sounds good.